### PR TITLE
STAR-318: Allowing a footer in the popup components

### PR DESCRIPTION
**Pull request by DanielSchiavini**
Issue: https://jira.ultimaker.com:8443/browse/STAR-318
Used in: https://github.com/Ultimaker/stardust-marketplace_portal/pull/144
This can be used to display errors in the popup.

### CODE REVIEW

File: src/stories/modal.stories.tsx [hunk 1]
```diff
@@ -49,15 +49,19 @@ stories.add('Confirm popup', withInfo(
...
 ));
 
 stories.add('Basic popup', withInfo(
     'PopupBase modal'
 )(() =>
     <PopupBase
         headerText="Basic popup"
-        width={selectV2('Popup width', widthOptions, widthDefaultValue)}>
+        width={selectV2('Popup width', widthOptions, widthDefaultValue)}
+        footer={text("Footer")}
```
Comment (woosie10) [inline src/stories/modal.stories.tsx]: single quotes

File: src/stories/modal.stories.tsx [hunk 2]
```diff
@@ -80,7 +84,9 @@ stories.add('Prompt popup', withInfo(
         validationHandler={validation}
         inputType="number"
         inputMin={1}
-        inputMax={100} />
+        inputMax={100}
+        footer={text("Footer")}
```
Comment (woosie10) [inline src/stories/modal.stories.tsx]: single quotes

File: src/stories/modal.stories.tsx [hunk 3]
```diff
@@ -106,7 +112,9 @@ stories.add('Multi-step popup', withState({ step: 1 })
                 secondaryBtnStyle="quiet"
                 step={store.state.step}
                 totalSteps={3}
-                width={selectV2('Popup width', widthOptions, widthDefaultValue)} />
+                width={selectV2('Popup width', widthOptions, widthDefaultValue)}
+                footer={text("Footer")}
```
Comment (woosie10) [inline src/stories/modal.stories.tsx]: single quotes

**Comment by ChrisTerBeke** [comment]
Linter time :P

**Pull request merged**